android/src/main/java/com/twiliovoicereactnative/AudioSwitchManager.java

```diff
@@ -90,22 +83,12 @@ private AudioSwitchManager(Context context) {
       if (this.listener != null) {
         this.listener.apply(audioDevices, selectedAudioDeviceUuid, selectedDevice);
       }
-
-      return null;
+      return Unit.INSTANCE;
```

Comment:
Just curious, what's the difference between the `Unit` type and `null` here? How does AudioSwitch behave differently?

Reply:
Oh Audioswitch is written in Kotlin and thats the expected return value in kotlin.... Unit type kinda maps to void, but only when the jvm allows it...

https://stackoverflow.com/questions/55953052/kotlin-void-vs-unit-vs-nothing